### PR TITLE
Spm default v2

### PR DESCRIPTION
**Pull request by remiadon**
Going from the simple first example on spm_default to a more complete example,
including all steps described in the batch.m file

TODO :
- [x] make sure we have 14 singular activities, which is the number we see in the left panel when opening the batch.m file via the SPM fmri GUI

### CODE REVIEW

File: examples/default/realign.json [hunk 1]
```diff
@@ -13,48 +13,79 @@
...
+        {
+          "@id" : "653467354367537gjsfhdgfhfdsjkn",
+          "@type" : "prov:SoftwareAgent",
+          "RRID" : "https://scicrunch.org/resources/Any/record/nlx_144509-1/SCR_007037/resolver?q=SPM&l=SPM",
+          "label" : "SPM" 
+        }
+      ],
       "prov:Activity" : [
         {
           "@id" : "realign",
```
Comment (cmaumet) [inline examples/default/realign.json]: To be closer to a real-world example, can you use an alphanumeric id (uuid) and keep the name in a "label" attribute?

File: examples/default/realign.json [hunk 2]
```diff
@@ -13,48 +13,79 @@
...
-            ]
-          }
+          "@id" : "sdfsdfjoisjdfojsojdofjiosdf",
+          "label" : "segment",
+          "wasAssociatedWith" : "653467354367537gjsfhdgfhfdsjkn",
+          "startedAtTime" : "10/10/2020 00:00:00",
+          "endedAtTime" : "10/10/2020 01:09",
+          "used" : "TODO",
+          "attributes" : [
+            ["tissue1.tpm", "$HOME/spm12/tpm/TPM.nii,1"],
```
[the remark this answers is not in the document]
Comment (cmaumet) [reply examples/default/realign.json]: Discussed w/ @remiadon on March 30. We'll keep the path to template as a string for now.

[the remark this answers is not in the document]
Comment (cmaumet) [reply examples/default/realign.json]: Discussed w/ @remiadon on March 30. We'll keep the standard table of "key, pair" for now.

**Comment by remiadon** [comment]
Missing activities : 
- 2x "move to" after "unzip", they have been merged, see `copies.json`
- "model specification" and "model estimation" have been merged

**Comment by remiadon** [comment]
@cmaumet ready to merge

**Pull request merged**